packages/quantic/force-app/main/default/lwc/quanticResultChildren/quanticResultChildren.js

```diff
@@ -173,13 +174,10 @@ export default class QuanticResultChildren extends LightningElement {
   }
 
   get shouldDisplayNoChildrenMessage() {
-    if (!this.isFirstLevelChildCollection) {
+    if (!this.isFirstLevelChildCollection || !this.areAllChildResultsLoaded) {
       return false;
     }
-    if (this.areAllChildResultsLoaded) {
-      return !this.moreResultsFound;
-    }
-    return !this.areMoreResultsAvailable;
+    return !this.moreResultsFound;
```

Comment:
Whenever a result doesn't even have a collection, this won't work no?

If a result doesn't even have a collection `this.moreResultsFound` should be `false`, so this will return `true`?

Reply:
If a result doesn't even have a collection `this.areAllChildResultsLoaded` will always be `false`. It means that we will always  return `false` following the first block of the condition.

Reply:
I think it's a bit confusing the way these conditions are paired now, essentially would this make sense instead?

```
if (!this.isFirstLevelChildCollection) {
    return false;
}
return this.areAllChildResultsLoaded && !this.moreResultsFound;
```

Reply:
Yeah I think this is more clear IMO since moreResultsFound is tied to the loading